util/misc.go

```diff
@@ -193,6 +193,8 @@ var (
 	PerformanceSchemaName = model.NewCIStr("PERFORMANCE_SCHEMA")
 	// MetricSchemaName is the `METRICS_SCHEMA` database name.
 	MetricSchemaName = model.NewCIStr("METRICS_SCHEMA")
+	// SysSchemaName is the `sys` database name.
+	SysSchemaName = model.NewCIStr("sys")
```

Comment:
If some users have used `sys` as the database name before, how to solve it here? Should we consider this situation? @qw4990 @rebelice

Reply:
Good question, I didn’t think about it before. I want to know how the information_schema or MySQL 8.0 was introduced at that time. Do you have any advice？ @qw4990